### PR TITLE
Use xcube / xcube-cci GH master branch

### PR DESCRIPTION
**Pull request by forman**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -10,11 +10,25 @@ install:
...
+  - git clone https://github.com/dcs4cop/xcube.git
+  - git clone https://github.com/dcs4cop/xcube-cci.git
+  # Install xcube env and xcube
+  - cd xcube
+  - mamba env create
+  - conda activate xcube
+  - python setup.py develop
+  # Install xcube-cci
+  - cd ..\xcube-cci
+  - mamba install -c conda-forge lxml pydap asyncio nest-asyncio
```
Comment (TonioF) [inline appveyor.yml]:
```suggestion
  - mamba install -c conda-forge lxml pydap aiohttp nest-asyncio
```

**Review by TonioF — CHANGES_REQUESTED** [review]
Please change

File: appveyor.yml (hunk 1 above)
Comment (forman) [reply appveyor.yml]: Hey, this is a draft!

Comment (forman) [reply appveyor.yml]: Already done.

**Review by TonioF — APPROVED** [review]

**Pull request merged**